crates/testenv/Cargo.toml

```
@@ -17,9 +17,16 @@ workspace = true
 
 [dependencies]
 bdk_chain = { path = "../chain", version = "0.20.0", default-features = false }
-electrsd = { version = "0.28.0", features = [ "bitcoind_25_0", "esplora_a33e97e1", "legacy" ] }
+electrsd = { version = "0.28.0", features = [ "legacy" ], default-features = false }
+
+[dev-dependencies]
+bdk_testenv = { path = "." }
 
 [features]
-default = ["std"]
+default = ["std", "download"]
+download = ["electrsd/bitcoind_25_0", "electrsd/esplora_a33e97e1"]
```

We probably don't need a new named feature per se, but instead just include the desired features in "default". What do you think?

```suggestion
default = ["std", "electrsd/bitcoind_25_0", "electrsd/esplora_a33e97e1"]
```

---

personally I think it's a bit more clear what's going on with the explicit 'download' feature. I would even add a comment maybe since this issue is a bit hard to parse and the solution isn't super direct (it's not super obvious that the purpose of the download feature is to allow disabling downloads in doc builds indirectly by disabling default features)